<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNoteForm</class>
 <widget class="QMainWindow" name="CreateNoteForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New note</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/resources/notebook--plus.png</normaloff>:/icons/resources/notebook--plus.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTableView" name="tv_Tags">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>18</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>18</number>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_Title">
        <property name="frame">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_NoteHtmlText">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="autoFormatting">
         <set>QTextEdit::AutoAll</set>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="documentTitle">
         <string/>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>3</number>
        </property>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="le_Tags">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Comma delimited tags:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pb_Cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pb_Ok">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Finish: Ctrl + Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionDecFontSize"/>
   <addaction name="actionIncFontSize"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/edit-bold.png</normaloff>:/icons/resources/edit-bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/edit-italic.png</normaloff>:/icons/resources/edit-italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/edit-underline.png</normaloff>:/icons/resources/edit-underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/arrow-turn-180-left.png</normaloff>:/icons/resources/arrow-turn-180-left.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/arrow-turn.png</normaloff>:/icons/resources/arrow-turn.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionIncFontSize">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/edit-size-up.png</normaloff>:/icons/resources/edit-size-up.png</iconset>
   </property>
   <property name="text">
    <string>IncFontSize</string>
   </property>
  </action>
  <action name="actionDecFontSize">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/resources/edit-size-down.png</normaloff>:/icons/resources/edit-size-down.png</iconset>
   </property>
   <property name="text">
    <string>DecFontSize</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>le_Title</tabstop>
  <tabstop>te_NoteHtmlText</tabstop>
  <tabstop>le_Tags</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>te_NoteHtmlText</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>te_NoteHtmlText</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionItalic</sender>
   <signal>toggled(bool)</signal>
   <receiver>te_NoteHtmlText</receiver>
   <slot>setFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUnderline</sender>
   <signal>toggled(bool)</signal>
   <receiver>te_NoteHtmlText</receiver>
   <slot>setFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_Tags</sender>
   <signal>editingFinished()</signal>
   <receiver>CreateNoteForm</receiver>
   <slot>adjustTags()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Ok</sender>
   <signal>clicked()</signal>
   <receiver>CreateNoteForm</receiver>
   <slot>finish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateNoteForm</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>adjustButtons(QTextCharFormat)</slot>
  <slot>setBold(bool)</slot>
  <slot>incFontSize()</slot>
  <slot>decFontSize()</slot>
  <slot>fontColorPopup()</slot>
  <slot>setHeader()</slot>
  <slot>adjustTags()</slot>
  <slot>finish()</slot>
  <slot>cancel()</slot>
 </slots>
</ui>
